### ISSUE TITLE
TAG_ARTIST should be moved to ExifIFD0Directory

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Rustemt/metadata-extractor#30]
```
Based a few test samples I've ran and also according to the sample output of 
FUJIFILM - FinePixS1Pro [1], I suppose the artist information (0x013b) should 
be part of the ExifIFD0Directory instead of the ExifSubIFDDirectory.


```

Original issue reported on code.google.com by `irq...@gmail.com` on 13 Jul 2011 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
Thanks for this report.  It seems you are indeed correct!  I'll double check 
whether I missed migrating any other tags when I split the old ExifDirectory 
into ExifIFD0 and ExifSubIFD directory classes.
```

Original comment by `drewnoakes` on 18 Jul 2011 at 4:23
- Changed state: **Started**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.5.0 RC3
```

Original comment by `drewnoakes` on 18 Nov 2011 at 10:36
- Changed state: **Fixed**
